src/backend/marsha/core/serializers/video.py

```diff
@@ -636,13 +636,16 @@ def get_live_info(self, obj):
         if obj.live_state is None or can_return_live_info is False:
             return {}
 
-        live_info = {
-            "medialive": {
-                "input": {
-                    "endpoints": obj.live_info["medialive"]["input"]["endpoints"],
-                }
-            },
-        }
+        live_info = {}
+
+        if obj.live_info is not None:
+            live_info = {
+                "medialive": {
+                    "input": {
+                        "endpoints": obj.live_info["medialive"]["input"]["endpoints"],
+                    }
+                },
+            }
```

Comment:
I find this weird. Why not return the whole obj.live_info since the method is called `get_live_info`? It seems that you are only interested in the live endpoints so it should be called `get_live`endpoints`?

Reply:
To not expose sensitive data stored in `live_info`. If I change the method name I also have to rename the exposed property and this will not match the video model any more.

Reply:
the exposed property could have a different name than the method and the method could be named something like `get_restricted_live_info` to convey the fact that it is live_info but truncated?

Reply:
as discussed, let's not rename the method and just add a comment to clarify why we restrict what is returned.